src/main/resources-filtered/org/obsidiantoaster/generator/ui/starter/plugin-template.xml.ftl

```diff
@@ -10,7 +10,7 @@
          <plugin>
             <groupId>io.fabric8</groupId>
             <artifactId>fabric8-maven-plugin</artifactId>
-            <version>3.1.92</version>
+            <version>${version.fabric8-maven-plugin}</version>
```

Comment:
the property is replaced during the build of the addon; so its never a maven property in the template thats actually used by the addon